extensions.yaml

```diff
@@ -5,6 +5,7 @@
 repos:
   - rhel-8-advanced-virt
   - rhel-8-nfv
+  - rhel-8-rhocp
```

Comment:
IOW it should work to just drop this line.

Reply:
This is something I'm not exactly sure @cgwalters.  Apart from myself breaking the CI with my change in the `openshift/release` (sorry), the whole situation of where or when the `kata-containers` package will show up is still not clear to me.

I've reverted the changed on `openshift/release` as it was impacting other bits of the project, and I'll try to sync up with all the folks possibly involved into the release process.  For now, let me hold this one.

Reply:
Situation is finally solved, this line has been dropped and seems that we're good to go (huge thanks to the AOS-ART team!)